Examples/Example/Pods/FigmaExport/Release/figma-export.yaml

```diff
@@ -31,6 +31,8 @@ ios:
   xcassetsPath: "./Resources/Assets.xcassets"
   # Is Assets.xcassets located in the main bundle?
   xcassetsInMainBundle: true
+  # Is Assets.xcassets located in a swift package?
+  xcassetsInSwiftPackage: false
```

Comment:
As described earlier this parameter should be false by default. So just remove it.